src/Microsoft.ML.StandardTrainers/Standard/LogisticRegression/LbfgsPredictorBase.cs

```diff
@@ -71,20 +77,19 @@ public abstract class OptionsBase : TrainerInputBaseWithWeight
             public float StochasticGradientDescentInitilaizationTolerance = 0;
 
             /// <summary>
-            /// Features must occur in at least this many instances to be included
+            /// Determines whether to produce output during training or not.
```

Comment:
>Determines whether to produce output during training or not. [](start = 16, length = 60)

should it be more explicit: If set to true produces no output during training. 

Users can imply the correct usage from the parameter name, but why leave it to implying.

Reply:
added

---
In reply to: [268038266](https://github.com/dotnet/machinelearning/pull/3059#discussion_r268038266) [](ancestors = 268038266)